util/src/test-support/java/tech/pegasys/artemis/util/async/AsyncRunnerTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.util.async;
+
+import static tech.pegasys.artemis.util.async.SafeFuture.propagateResult;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+public class AsyncRunnerTest implements AsyncRunner {
+
+  public AsyncRunnerTest() {}
```

Comment:
A no-arg constructor is provided by default so this can be omitted.

Reply:
Done.